docs/planning/backend/app/scrapers/fragrantica_scraper.py

```
@@ -0,0 +1,312 @@
+
+                name = name_elem.get_text(strip=True) if name_elem else ""
+                brand = brand_elem.get_text(strip=True) if brand_elem else ""
+
+                if name and href:
+                    results.append({
+                        'name': name,
+                        'brand': brand,
+                        'url': href
+                    })
+            except Exception:
+                continue
```

Ruff (0.14.10)

`try`-`except`-`continue` detected, consider logging the exception

(S112)